generate_rospkg_apkbuild/genapkbuild.py

```diff
@@ -252,19 +252,18 @@ def package_to_apkbuild(ros_distro, package_name,
         sys.exit(1)
 
     # Remove duplicated dependency keys
-    depends_keys = list(set(depends_keys))
-    depends_export_keys = list(set(depends_export_keys))
-    makedepends_keys = list(set(makedepends_keys))
+    depends_keys = sorted(list(set(depends_keys)))
+    depends_export_keys = sorted(list(set(depends_export_keys)))
+    makedepends_keys = sorted(list(set(makedepends_keys)))
```

Comment:
Sets doesn't seems to fit your needs as the order is important.
What about `list(collections.OrderedDict.fromkeys(...))`?

Reply:
Using `OrderedDict` sounds good to keep original order.
`catkin_pkg.package.parse_package_string` seems to store dep keys by the same order in package.xml.
So, it would be good to sort keys explicitly to keep APKBUILD constant even if orders in package.xml is changed.

(This PR is intended to stop shuffling keys like https://github.com/seqsense/aports-ros-experimental/pull/73/files.)

Reply:
That makes sense now. Thanks.